lib/sinon/call.js

```diff
@@ -179,8 +174,8 @@ var callProto = {
             }
         }
         if (this.stack) {
-            callStr += this.getStackFrames()[0].replace(/^\s*(?:at\s+|@)?/, " at ");
```

Comment:
I inlined the `getStackFrames` function call, because I need all the `toString` logic to be self-contained in order to call it using a fake `spy` object (see https://github.com/sinonjs/sinon/pull/1188/files#diff-28fe4d08750ea6ea89c7d94be8aaa928R186)